File: Hjemmesentral/app/src/main/java/com/homecentral/jrs/hjemmesentral/model/yr/Forecast.java
package com.homecentral.jrs.hjemmesentral.model.yr;

import com.homecentral.jrs.hjemmesentral.model.yr.forecast.Time;

import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.util.List;

@Root(name = "forecast", strict = false)
public class Forecast {

    @ElementList(name="tabular")
    private List<Time> tabular;

    public List<Time> getTabular() {
        return tabular;
    }

    public void setTabular(List<Time> tabular) {
        this.tabular = tabular;
    }
}

File: Hjemmesentral/app/src/main/java/com/homecentral/jrs/hjemmesentral/model/yr/Forecast.kt
package com.homecentral.jrs.hjemmesentral.model.yr

import com.homecentral.jrs.hjemmesentral.model.yr.forecast.Time
import org.simpleframework.xml.ElementList
import org.simpleframework.xml.Root

@Root(name = "forecast", strict = false)
class Forecast @JvmOverloads constructor(
    @field:ElementList(name = "tabular") var tabular: List<Time>? = null
)